ios/Plugin/Plugin.swift

```diff
@@ -12,11 +12,11 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
         Radar.setDelegate(self)
     }
 
-    public func didReceiveEvents(_ events: [RadarEvent], user: RadarUser) {
+    public func didReceiveEvents(_ events: [RadarEvent], user: RadarUser?) {
         DispatchQueue.main.async {
             self.notifyListeners("events", data: [
                 "events": RadarEvent.array(for: events) ?? [],
-                "user": user.dictionaryValue()
+                "user": user?.dictionaryValue() ?? []
```

Comment:
since this is a dictionary value, wondering if it makes sense to default to {} rather than []

Reply:
Is this the stuff that gets displayed in the pop-up dialog when events are received? If so, I'm not sure whether the type matters, even though it does feel a little weird to have different types on both sides of the `??`.

Reply:
I changed all of them to return `{}`. Let me know if I missed any.